Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Description: Tighten dependency on env_logger, since buildds choose the first of several build-deps and 0.11 is not in debian yet.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -70,7 +70,7 @@
 ]
 
 [dev-dependencies.env_logger]
-version = ">=0.10, <0.12"
+version = "0.10"
 
 [dev-dependencies.sequoia-openpgp]
 version = "1"
